openni2_camera/include/openni2_camera/openni2_device.h

```diff
@@ -160,9 +160,6 @@ class OpenNI2Device
   bool depth_video_started_;
 
   bool image_registration_activated_;
-
-  bool use_device_time_;
```

Comment:
this is unused - removing it does break ABI - but these packages haven't even been synced out yet